### PR TITLE
Update Management Console architecture page to include new Management Updater

### PR DESCRIPTION
**Pull request by ricci2511**
# Description

This PR updates the architecture page of the Management console to include the new Management Updater.

## Related issues

- Closes https://github.com/united-manufacturing-hub/MgmtIssues/issues/1458

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [ ] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

## Additional information

The actual diagram of the page still needs to be updated. I didn't find a way to export the structurizr dsl to svg/png to update the current image. So I would appreciate if it could be done by any of you.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/architecture/_index.md [hunk 1]
```diff
@@ -61,12 +61,14 @@ workspace {
...
             redisMgmt = container "Message Queue" "Stores and forwards messages" "Redis"
-            
+            upgrader = container "Upgrader" "Upgrades UMH instances" "Docker, Go"
+
             spa -> backend "Sends/receives E2E encrypted messages, fetches encrypted private key from backend"
             backend -> keyStorage "Fetches encrypted private key"
             backend -> companion "Sends/receives E2E encrypted messages"
             backend -> redisMgmt "Stores and fetches E2E encrypted messages"
-            
+            backend -> upgrader "Sends/receives E2E encrypted messages"
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/architecture/_index.md:70]: i don't think this architecture is correct

File: umh.docs.umh.app/content/en/docs/architecture/management-console/_index.md [hunk 2]
```diff
@@ -29,6 +29,15 @@ for decrypting messages coming from the user via the Backend and executing
 requested actions. Responses are end-to-end encrypted as well, maintaining a
 secure and opaque channel to the Backend.
 
+## Management Updater
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/architecture/management-console/_index.md:32]: You talk about updater and upgrader. We need to be consistent with the wording.

File: umh.docs.umh.app/content/en/docs/architecture/management-console/_index.md [hunk 3]
```diff
@@ -29,6 +29,15 @@ for decrypting messages coming from the user via the Backend and executing
...
+
+The Updater is an essential component integrated into the Management Console, designed to streamline the upgrading
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/architecture/management-console/_index.md:34]: This is really a lot of ChatGPT. One can read the text, but still have no idea what is going on. Can you remove all the filler words?

File: umh.docs.umh.app/content/en/docs/architecture/_index.md [hunk 4]
```diff
@@ -280,8 +282,8 @@ mechanisms.
 
 {{< mermaid theme="neutral" >}}
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/architecture/_index.md:283]: also you need to generate an SVG / PNG from it so your changes are also shown to the user

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/architecture/_index.md (hunk 4 above)
Comment (ricci2511) [reply umh.docs.umh.app/content/en/docs/architecture/_index.md:283]: See the "Additional information" section of the PR's description

**Pull request merged**